app/Http/Controllers/ContactMessageController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Mail;
+use Illuminate\Http\Request;
+use App\ContactMessage;
+
+class ContactMessageController extends Controller
+{
+    /**
+     * 
+     */
+    public function index()
+    {
+        return view('contact');
+    }
+
+    /**
+     * Sends an email to the JeepMeet admin and
+     * stores the visitor/user contact message
+     * to the database.
+     * 
+     * @param $request - Request object
+     * 
+     * @return Illuminate\Routing\Redirector
+     */
+    public function store(Request $request)
+    {
+        $this->validate($request, [
+            'first_name' => 'required',
+            'email'      => 'required|email',
+            'message'    => 'required'
+        ]);
+
+        try {
+            // Send an email to jeepmeetup@gmail.com about a received Contact Us message
+            // sendMailToAdmin($request);
```

Comment:
This line shouldn't be commented. The function call send the email to the JeepMeet admins.